### PR TITLE
Docs Updates to Follow Style Guides. 

### PR DESCRIPTION
**Pull request by kaitlincart**
Updates the install and upgrade pages to adhere to our style guides.

### CODE REVIEW

File: website/content/docs/installation/index.mdx [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
   package using standard procedures for your operating system.
 ---
 
-# Installing Vagrant
+# Install Vagrant
 
-Installing Vagrant is extremely easy. Head over to the
-[Vagrant downloads page](/downloads) and get the appropriate installer or
-package for your platform. Install the package using standard procedures for
+To get started with Vagrant, you will need to download the appropriate installer or
```
Comment (tunzor) [inline website/content/docs/installation/index.mdx:11]:
```suggestion
To get started with Vagrant, download the appropriate installer or
```

File: website/content/docs/installation/index.mdx [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
+package for your platform from our
+[Vagrant downloads page](/downloads). Install the package with the standard procedures for
 your operating system.
 
-The installer will automatically add `vagrant` to your system path
-so that it is available in terminals. If it is not found, please try
-logging out and logging back in to your system (this is particularly
-necessary sometimes for Windows).
+The installer should automatically add `vagrant` to your system path
```
Comment (tunzor) [inline website/content/docs/installation/index.mdx:16]:
```suggestion
The installer automatically adds `vagrant` to your system path
```

File: website/content/docs/installation/index.mdx [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+a [box](https://developer.hashicorp.com/vagrant/tutorials/getting-started/getting-started-boxes).
 
-Sometimes, certain hypervisors do not allow you to bring up virtual machines
-if more than one hypervisor is in use. If you are lucky, you might see the following
-error message come up when trying to bring up a virtual machine with Vagrant and
-VirtualBox:
+## How to use multiple hypervisors
 
-```text
+If you have more than one hypervisor is in use, hypervisors often do not allow you to bring up virtual machines.
```
Comment (tunzor) [inline website/content/docs/installation/index.mdx:33]:
```suggestion
Hypervisors often do not allow you to bring up virtual machines if you have more than one hypervisor in use.
```

File: website/content/docs/installation/index.mdx [hunk 4]
```diff
@@ -50,16 +50,9 @@ VBoxManage: error: VirtualBox can't operate in VMX root mode. Please disable the
...
 
-Other operating systems like Windows will blue screen if you attempt to bring up
-a VirtualBox VM with Hyper-V enabled. Below are a couple of ways to ensure you
-can use Vagrant and VirtualBox if another hypervisor is present.
-
-### Linux, VirtualBox, and KVM
-
-The above error message is because another hypervisor (like KVM) is in use.
-We must blacklist these in order for VirtualBox to run correctly.
+You must denylist the additional hypervisors in order for VirtualBox to run correctly.
```
Comment (tunzor) [inline website/content/docs/installation/index.mdx:53]:
```suggestion
You must add the additional hypervisors to the deny list in order for VirtualBox to run correctly.
```
`Denylist` as a verb sounds odd to me but feel free to keep it as I learned today that it is, in fact, a verb 😅

File: website/content/docs/installation/index.mdx [hunk 5]
```diff
@@ -68,35 +61,39 @@ kvm                   593920  1 kvm_intel
 irqbypass              16384  1 kvm
 ```
 
-The one we're interested in is `kvm_intel`. You might have another.
-
-Blacklist the hypervisor (run the following as root):
+Use the `blacklist` command to add the hypervisor to you denylist.
```
Comment (tunzor) [inline website/content/docs/installation/index.mdx:64]:
```suggestion
Use the `blacklist` command to add the hypervisor to your denylist.
```

File: website/content/docs/installation/upgrading.mdx [hunk 6]
```diff
@@ -1,31 +1,34 @@
...
 [continue to work](/docs/installation/backwards-compatibility),
-newer Vagrantfiles may have backwards incompatible changes until 2.0 final.
+newer Vagrantfiles may have backwards incompatible changes until 3.0.
+
+## Issue reports
 
--> **Run into troubles upgrading?** Please [report an issue](https://github.com/hashicorp/vagrant/issues)
-if you run into problems upgrading. Upgrades are meant to be a smooth
-process and we consider it a bug if it was not.
+If you encounter any problems at upgrade time, [report them issue in Github](https://github.com/hashicorp/vagrant/issues). Upgrades are meant to be a smooth process and we consider it a bug if it was not.
```
Comment (tunzor) [inline website/content/docs/installation/upgrading.mdx:34]:
```suggestion
If you encounter any problems at upgrade time, [report them as an issue in Github](https://github.com/hashicorp/vagrant/issues). Upgrades are meant to be a smooth process and we consider it a bug if it was not.
```

**Review by tunzor — CHANGES_REQUESTED** [review]
Just a few language changes but otherwise LGTM 👍

**Review by tunzor — APPROVED** [review]
👍

**Review by soapy1 — APPROVED** [review]

**Pull request merged**